src/client/components/UserHeader.less

```diff
@@ -69,6 +69,18 @@
   padding: 30px 16px;
   align-items: center;
   flex-direction: column;
+  a {
+    position: relative;
+  }
+  &--active {
+    position: absolute;
+    left: 75px;
+    bottom: 0;
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+    background: #a8f460;
```

Comment:
We have a var set for the green color `@green` can you use it instead?

Reply:
The variable `@green` is not the same color of the screenshot you provided. Is that normal / OK ?

Reply:
Yes it's fine @gregory-latinier i've used a random color in the screenshot.